pri/boostdetect.pri

```diff
@@ -13,44 +13,55 @@
 # along with Fritzing. If not, see <http://www.gnu.org/licenses/>.
 # ********************************************************************/
 
+LATESTBOOST = 0
 
-exists($$boost_root) {
-    INCLUDEPATH += $$absolute_path($$boost_root)
-    message("using boost $$absolute_path($$boost_root)")
+# reduce warning messages by checking if variables are defined before using them
+defined(boost_root, var) {
+    LATESTBOOST = user
+    exists($$boost_root) {
+        # force absolute: relative would break due to detection versus use folder nest level
+        BOOSTPATH = $$absolute_path($$boost_root)
+        absolute_boost = 1
+        unset(absolute_boost)
```

Comment:
Why assign it and then unset?

Reply:
Easier than checking if exists then unsetting. Safety measure, in case it was set in the calling .pro file. Did not want it to accidentally carry through to the end, where it would break the absolute test and set.